src/options.js

```diff
@@ -5,4 +5,13 @@ export default {
     platformSpeedIncrement: 0,
     windowHeight: 600,
     windowWidth: 800,
+    purpleBox: 0x6400e4,
```

Comment:
Is there a reason some colors are in hex format (`0x6400e4`) and some are strings (`#6400e4`)? If not, let's standardize on strings. TIL you can use hex codes as colors!

Reply:
IIRC, Phaser has some funky/inconsistent means of adding color. But agreed, if there's a way to be consistent across our styling, I'm all for it :)

Reply:
Yeah, rectangle requires the color to be a number (0x000000) while text requires it to be a string("#00000")

<img width="1570" alt="Screen Shot 2019-07-03 at 10 19 55 AM" src="https://user-images.githubusercontent.com/26859409/60599278-30b22e80-9d7c-11e9-9d50-f78e8e9fa04a.png">

Reply:
I was wrong. Cool 😄